Makefile

```diff
@@ -38,8 +38,23 @@ test:
 	go test -i $(TEST) || exit 1
 	echo $(TEST) | xargs -t -n4 go test $(TESTARGS) -timeout=30s -parallel=4
 
+.PHONY: infra.reset
+infra.reset: infra.down infra.up ## Reset localstack infra
+
+.PHONY: infra.down
+infra.down: ## Destroy localstack infra
+	-cd localstack && terraform destroy -auto-approve
+	-cd localstack && rm terraform.tfstate*
+
+.PHONY: infra.up
+infra.up: ## Setup localstack for development
```

Comment:
to get this working I had to change `http://localstack....` to `http://localhost`. I didn't check it in for now, but with that change this all works for me. Just FYI.

Reply:
FYI, you can update you `/etc/hosts` file and add an entry for `localstack` to be `127.0.0.1` (the loopback address). This makes it so if you're running in docker-compose and using localstack, the DNS name localstack works (goes to the container named localstack), and if you're running in your shell terminal localstack still just goes back to your own system.